cpmpy/expressions/globalconstraints.py

```diff
@@ -387,3 +387,31 @@ def deepcopy(self, memodict={}):
        """
         copied_args = self._deepcopy_args(memodict)
         return Xor(copied_args)
+
+class NativeConstraint(Expression):
```

Comment:
Don't we want to subclass `GlobalConstraint` for this? Then current transformations will be able to work as is

Reply:
No, because then the `decompose()` function should be supported, while the idea is that this can only be used directly.

transformations should continue to work though... it should handle any expression (e.g. leave these as is)